openjdk/build.xml

```diff
@@ -93,13 +84,30 @@
 				<property name="regressionRepo" value="https://github.com/AdoptOpenJDK/${jdkName}.git" />
 			</else>
 		</if>
-
-		<echo message="git clone --depth 1 -q ${gitReleaseTag} ${regressionRepo}" />
-		<exec executable="git" failonerror="false">
-			<arg line="clone --depth 1 -q ${gitReleaseTag} ${regressionRepo}" />
-		</exec>
+		<if>
+			<isset property="env.RELEASE_TAG"/>
+			<then>
+				<exec executable="git" failonerror="false">
```

Comment:
Hmm, wondering what are the consequences on not failing if there is an error to clone this repo?  I know it predates your changes, but should we change to failonerror="true" (and same for git clone command below)?

Reply:
Default failonerror is false. Definitely we should explicitly set it as true if git clone exits with error. Updated.
FYI @Mesbah-Alam should system-tests do the same update?